src/Bridge/Symfony/DependencyInjection/FidryAliceDataFixturesExtension.php

```diff
@@ -44,66 +43,51 @@ public function load(array $configs, ContainerBuilder $container)
         $processedConfiguration = $this->processConfiguration($configuration, $configs);
         $bundles = array_flip($container->getParameter('kernel.bundles'));
 
-        if (false === array_key_exists(self::NELMIO_ALICE_BUNDLE_CLASS, $bundles)) {
+        if (false === array_key_exists(NelmioAliceBundle::class, $bundles)) {
             throw new \LogicException(
                 sprintf(
                     'Cannot register "%s" without "%s".',
                     FidryAliceDataFixturesBundle::class,
-                    self::NELMIO_ALICE_BUNDLE_CLASS
+                    NelmioAliceBundle::class
                 )
             );
         }
 
         $loader = new XmlFileLoader($container, new FileLocator(self::SERVICES_DIR));
         $loader->load('loader.xml');
 
-        $this->registerDoctrineORMConfig($bundles, $processedConfiguration, $loader);
-        $this->registerEloquentConfig($bundles, $processedConfiguration, $loader);
+        $this->registerConfig(Configuration::DOCTRINE_ORM_DRIVER, DoctrineBundle::class, $bundles, $processedConfiguration, $loader);
+        $this->registerConfig(Configuration::DOCTRINE_MONGODB_ODM_DRIVER, DoctrineMongoDBBundle::class, $bundles, $processedConfiguration, $loader);
+        $this->registerConfig(Configuration::DOCTRINE_PHPCR_ODM_DRIVER, DoctrinePHPCRBundle::class, $bundles, $processedConfiguration, $loader);
+        $this->registerConfig(Configuration::ELOQUENT_ORM_DRIVER, WouterJEloquentBundle::class, $bundles, $processedConfiguration, $loader);
     }
 
-    private function registerDoctrineORMConfig(array $bundles, array $configs, LoaderInterface $loader)
-    {
-        /** @var bool|null $isEnabled */
-        $isEnabled = $configs['db_drivers'][Configuration::DOCTRINE_ORM_DRIVER];
-        if (false === $isEnabled) {
-            return;
-        }
-
-        $doctrineBundleIsRegistered = array_key_exists(self::DOCTRINE_ORM_BUNDLE_CLASS, $bundles);
-        if ($isEnabled && false === $doctrineBundleIsRegistered) {
-            throw new \LogicException(
-                sprintf(
-                    'Cannot enable doctrine driver as the bundle "%s" is missing',
-                    self::DOCTRINE_ORM_BUNDLE_CLASS
-                )
-            );
-        }
-
-        if ($doctrineBundleIsRegistered) {
-            $loader->load('doctrine_orm.xml');
-        }
-    }
-
-    private function registerEloquentConfig(array $bundles, array $configs, LoaderInterface $loader)
-    {
+    private function registerConfig(
```

Comment:
a quick phpdoc is missing here as well to specify what `$bundles` and `$configs` are